monitoring/src/index.ts

```diff
@@ -24,6 +24,15 @@ export const io: SocketIOServer = new SocketIOServer(server, {
   }
 })
 
+io.use(function (socket, next): void {
+  if (socket.handshake.query && socket.handshake.query.token) {
+    console.log('middleware socket validation: ' + socket.handshake.query.token)
+    if (jwtManager.verify(socket.handshake.query.token as string)) next()
+  } else {
+    next(new Error('Authentication error'))
+  }
+})
```

Comment:
Nice!

Reply:
it is not a test log but i followed the idea kept also in the other authentication middleware (where we print auth tokens). If you think that is better to remove this log for the sockets i can remove it without any problem :D